homeassistant/components/remote_calendar/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: exempt
+    comment: |
+      No unique identifier.
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: |
+      There are no actions.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: |
+      No actions available.
+  brands: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow:
+    status: exempt
+    comment: |
+      There is no authentication required.
+  parallel-updates: done
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: no configuration options
+
+  # Gold
+  devices: done
+  diagnostics:
+    status: todo
+    comment: Diagnostics not implemented, yet.
+  discovery-update-info:
+    status: todo
+  discovery:
```

Comment:
Is there a discovery protocol for ics urls? I would assume these would be exempt since they don't really exist.

Reply:
No, I changed it here: https://github.com/home-assistant/core/pull/138862/commits/21cf5e83743d0539cbf27d332a74cecdeb522a0e